.github/workflows/build.yaml

```diff
@@ -111,7 +111,7 @@ jobs:
       - name: Install deps
         run: |
           sudo -E make deps
```

Comment:
thinking out loud,  :thinking: ... we could actually add the `cos` repository itself in `make deps` and consume the same binary in the tree